src/realm/util/file.cpp

```diff
@@ -633,8 +640,15 @@ void File::resize(SizeType size)
         size = data_size_to_encrypted_size(size);
 
     seek(size);
```

Comment:
I assume this seek() will not fail even though the disk is full? Could be nice with a source code comment on this